ginko_ls/src/server.rs

```diff
@@ -10,17 +10,24 @@ use tower_lsp::lsp_types::*;
 use tower_lsp::{Client, LanguageServer};
 use url::Url;
 
+pub struct BackendOptions {
+    pub include_paths: Option<Vec<String>>,
```

Comment:
I generally think that a better design is to move this to the `LanguageServer::initialize` or `LanguageServer::did_change_configuration` function. That way, the include paths can come from the IDE, resp. language front-end. I think that most people won't use the language server binary but use it packaged in some extension for their IDE (i.e., the VSCode extension).
However, I'm not super keen on this change, just mark this conversation as resolved if you don't want to implement it – I will merge this PR nonetheless.